src/MoBi.UI/Views/SimulationExplorerView.cs

```diff
@@ -32,11 +38,16 @@ private void compareNodeValues(object sender, CompareNodeValuesEventArgs e)
          if (e.Node1.Level == 0)
             e.Result = 0;
 
+         //we need to sort the module nodes exactly as we do on the ModuleExplorerView.cs
+         if (nodeIsModuleNode(e.Node1.ParentNode))
+            e.Result = _moduleExplorerPresenter.OrderingComparisonFor(e.Node1.Tag as ITreeNode<IWithName>, e.Node2.Tag as ITreeNode<IWithName>);
```

Comment:
A better way to do this is to pull up the `OrderingComparisonFor` from the `ModuleExplorerPresenter` into the `ExplorerPresenter` that is common to both. 

Make sure this ordering is only used for the appropriate nodes in both cases.